### PR TITLE
Switch to discuss.istio.io

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: content/blog/2019/announcing-discuss.istio.io/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+features (for example, threading) meant it wasn’t ideal for any longer discussions around a single issue. It also led to a dilemma for
+some users -- when should I email istio-users@googlegroups.com and when should I use RocketChat?
+
+We think we’ve found the right balance of features in a single platform, and we’re happy to announce
+[discuss.istio.io](https://discuss.istio.io). It’s a full-featured forum where we will have discussions about Istio from here on out.
+It will allow you to ask a question and get threaded replies! As a real bonus, you can use your GitHub identity.
+
+If you prefer emails, you can configure it to send emails just like Google groups did.
+
+We will be marking our Google groups “read only” so that the content remains, but we ask you to send further questions over to
```
Comment (linsun) [inline content/blog/2019/announcing-discuss.istio.io/index.md]: what is the guideline for existing emails that still require an answer?  Should we ask users to repost to discuss.istio.io?

File: content_zh/help/ops/security/debugging-authorization/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-title: 调试授权 
-description: 展示如何调试授权功能。 
+title: 调试授权
+description: 展示如何调试授权功能。
 weight: 5
 keywords: [debug,security,authorization,RBAC]
 ---
 
-这篇文章展示了如何调试 Istio 的授权。如果在本文的指导下你还是没有找到解决方案。我们很乐意你向 `istio-security@googlegroups.com` 发送邮件以寻求帮助。
+这篇文章展示了如何调试 Istio 的授权。如果在本文的指导下你还是没有找到解决方案
```
Comment (linsun) [inline content_zh/help/ops/security/debugging-authorization/index.md]: missing a 。at the end.

File: content_zh/about/community/join/index.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ Istio 是一个开源项目，拥有一个支持其使用和持续开发的活
   <tbody>
 
     <tr>
-      <td class="logo"><img src="/about/community/join/mailing-list.svg" alt="Mailing Lists"/></td>
+      <td class="logo"><img src="/about/community/join/discourse.svg" alt="Discourse"/></td>
       <td class="desc">
-        加入 <a href="https://groups.google.com/forum/#!forum/istio-users-zh">istio-users-zh@</a> 邮件列表用中文参与讨论和获取帮助。
-        加入 <a href="https://groups.google.com/forum/#!forum/istio-announce">istio-announce@</a> 邮件列表以了解最新版本、活动和其他事件。
-        还有一个 <a href="https://groups.google.com/forum/#!forum/istio-dev">istio-dev@</a> 邮件列表用于讨论围绕 Istio 项目开发的问题。
+        Head for the <a href="https://discuss.istio.io">Istio discussion board</a> to participate in
```
Comment (linsun) [inline content_zh/about/community/join/index.md]: @rootsongjc can you submit a follow up PR to correct this?  We have a special category for Chinese users too in discuss.istio.io.

**Comment by linsun** [comment]
LGTM, just left a comment in the blog - hope it can be addressed.  If the answer is resend, we need to give that instruction. :)

File: content/blog/2019/announcing-discuss.istio.io/index.md (hunk 1 above)
Comment (geeknoid) [reply content/blog/2019/announcing-discuss.istio.io/index.md]: I added a sentence about needing to repost.

File: content_zh/about/community/join/index.md (hunk 3 above)
Comment (rootsongjc) [reply content_zh/about/community/join/index.md]: @linsun OK. I'll do it.

**Pull request merged**